### PR TITLE
[13.0][MIG] l10n_de_skr04

### PR DESCRIPTION
**Pull request by AMhadhbi**
Description of the issue/feature this PR addresses:
Please do the review, thanks
Current behavior before PR:
Nothing to do
Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/l10n_de_skr04/migrations/13.0.3.0/openupgrade_analysis_work.txt [hunk 1]
```diff
@@ -0,0 +1,41 @@
+---Models in module 'l10n_de_skr04'---
+---Fields in module 'l10n_de_skr04'---
+---XML records in module 'l10n_de_skr04'---
+NEW account.account.template: l10n_de_skr04.chart_skr04_1206
+account.chart.template: l10n_de_skr04.l10n_chart_de_skr04 (noupdate switched)
```
Comment (MiquelRForgeFlow) [inline addons/l10n_de_skr04/migrations/13.0.3.0/openupgrade_analysis_work.txt:5]: something to do here

**Review by MiquelRForgeFlow — APPROVED** [review]

File: addons/l10n_de_skr04/migrations/13.0.3.0/openupgrade_analysis_work.txt (hunk 1 above)
Comment (AMhadhbi) [reply addons/l10n_de_skr04/migrations/13.0.3.0/openupgrade_analysis_work.txt:5]: Done.

File: odoo/openupgrade/doc/source/modules120-130.rst [hunk 2]
```diff
@@ -273,7 +273,7 @@ missing in the new release are marked with |del|.
 +----------------------------------------------+-------------------------------------------------+
 |l10n_de_skr03                                 |                                                 |
 +----------------------------------------------+-------------------------------------------------+
-|l10n_de_skr04                                 |                                                 |
+|l10n_de_skr04                                 | Nothing to do                                   |
```
Comment (MiquelRForgeFlow) [inline odoo/openupgrade/doc/source/modules120-130.rst]: Done instead

Comment (AMhadhbi) [reply odoo/openupgrade/doc/source/modules120-130.rst]: Done.

**Review by MiquelRForgeFlow — APPROVED** [review]

**Pull request merged**